### PR TITLE
Release fix/254 

### LINKED ISSUE #254: Text in the input field for timer
**Describe the bug**
When creating a timer you can enter letters in the input fields. You are however not allowed to create the timer, but no feedback is given to the user. If you enter letters in one field and numbers in another, you can still create the timer.

**To Reproduce**
1. Go to an activity's page and add press the add timer button and enter some text in the field 
2. Click on the checkmark
3. No error is shown to the user

**Expected behavior**
Either a dialog should be shown to the user that they can't create a timer with text as input, or they should not be allowed to enter text at all

**Actual behavior**
Nothing really happens

This was done on an emulator

### PR DESCRIPTION
**Pull request by msimon16**
This closes #254 
When adding a timer, putting letters or other symbols in the textfields now promts with an error message, notifying what is allowed.

image

image

### CODE REVIEW

**Comment by User12837985123** [comment]
@aau-giraf/sw610f19 you have been assigned this. Here's your checklist. I marked @Moesgaarda as the reviewer, but you might change it as you like.
Please use the following checklist. Feel free to check it off as you go along.
**Code Design**
- [ ] Is the code in the right place? (Both in terms of folder structure and class structure)
- [ ] Could this code, to the best of your knowledge, have reused existing code?
- [ ] Is the code over-engineered? Examples of over-engineering: Implemented behavior for future problems
- [ ] Does it introduce functionality that is not necessary for solving the problem?

**Code Readability and Maintainability**
- [ ] Are names meaningful and self-documenting?
- [ ] Is it understandable, by reading the code, what it does?
- [ ] Is the code simple and readable? i.e. it contains no “hack”, or obscure solution.
- [ ] Is it understandable, by reading the tests, what they do?
- [ ] Do the tests cover sensible cases? Both happy paths and exception paths?
- [ ] Do the tests cover the full functionality? i.e., will the tests fail if some of the requested functionality is missing?
- [ ] Are error messages in the code understandable?
- [ ] Are confusing sections of code covered by comments?

**Functionality**
For this part, open the app, and test:
- [ ] Are the features that the code claims to implement, actually implemented?
- [ ] Does it look like the code has bugs?
- [ ] Are all new screens reachable through in-app navigation?

**Points for Discussion and Reflection**
- [ ] Are UI messages, that the user might meet, actually helpful?
- [ ] Are there errors that are bad practice? Like, hard-coding a test database connection?
- [ ] Is there well written code in this? Remember to tell the developer!

**Comment by User12837985123** [comment]
@aau-giraf/sw608f19 you have been assigned this. Here's your checklist. I marked @TrineHolmager as the reviewer, but you might change it as you like.
Please use the following checklist. Feel free to check it off as you go along.
**Code Design**
- [ ] Is the code in the right place? (Both in terms of folder structure and class structure)
- [ ] Could this code, to the best of your knowledge, have reused existing code?
- [ ] Is the code over-engineered? Examples of over-engineering: Implemented behavior for future problems
- [ ] Does it introduce functionality that is not necessary for solving the problem?

**Code Readability and Maintainability**
- [ ] Are names meaningful and self-documenting?
- [ ] Is it understandable, by reading the code, what it does?
- [ ] Is the code simple and readable? i.e. it contains no “hack”, or obscure solution.
- [ ] Is it understandable, by reading the tests, what they do?
- [ ] Do the tests cover sensible cases? Both happy paths and exception paths?
- [ ] Do the tests cover the full functionality? i.e., will the tests fail if some of the requested functionality is missing?
- [ ] Are error messages in the code understandable?
- [ ] Are confusing sections of code covered by comments?

**Functionality**
For this part, open the app, and test:
- [ ] Are the features that the code claims to implement, actually implemented?
- [ ] Does it look like the code has bugs?
- [ ] Are all new screens reachable through in-app navigation?

**Points for Discussion and Reflection**
- [ ] Are UI messages, that the user might meet, actually helpful?
- [ ] Are there errors that are bad practice? Like, hard-coding a test database connection?
- [ ] Is there well written code in this? Remember to tell the developer!

File: lib/widgets/giraf_activity_time_picker_dialog.dart [hunk 1]
```diff
@@ -143,10 +145,27 @@ class GirafActivityTimerPickerDialog extends StatelessWidget {
...
       _timerBloc.addTimer(duration);
       Routes.pop(context);
+    } else {
+      showDialog<Center>(
+          context: context,
+          barrierDismissible: false,
+          builder: (BuildContext context) {
+            return GirafNotifyDialog(
+                key: const Key('TimerWrongInputKey'),
+                title: 'Forkert input',
```
Comment (Tekrus) [inline lib/widgets/giraf_activity_time_picker_dialog.dart]:
```suggestion
                title: 'Ugyldigt input',
```

**Review by Tekrus — DISMISSED** [review]

**Comment by ChristianBroS** [comment]
It is still possible to enter letters in one field and numbers in another and still create the timer. Or is this the correct behavior?
Also if you put a negative number in (e.g. -1), you are met with the UI message "Den indtastede tid må ikke være 0".

**Comment by msimon16** [comment]
> It is still possible to enter letters in one field and numbers in another and still create the timer. Or is this the correct behavior?
> Also if you put a negative number in (e.g. -1), you are met with the UI message "Den indtastede tid må ikke være 0".

This is not expected behavior. Will be looking into it.

**Comment by msimon16** [comment]
> > It is still possible to enter letters in one field and numbers in another and still create the timer. Or is this the correct behavior?
> > Also if you put a negative number in (e.g. -1), you are met with the UI message "Den indtastede tid må ikke være 0".
> 
> This is not expected behavior. Will be looking into it.

This should be fixed now, pls review :)

**Review by ChristianBroS — APPROVED** [review]
Good work :)

**Review by Tekrus — APPROVED** [review]

**Pull request merged**